### ISSUE TITLE
Disabled sorters become inverted sorters. Use "configure" property to change state?

### ISSUE THREAD

**Issue by schuhwerk** [Anuken/Mindustry-Suggestions#3728]
### Describe what you would like changed, and why.

Thank you for this great game!

Tldr.: It seems unintuitive to me, that a disabled sorter becomes an inverted sorter. 

And the longer story:
Today I implementing an idea: Use the nice interface, which sorters give you, to avoid resources being destroyed (and ramp up production in other places [^1])
I added a Processor which loops through all connected sorters, "sensor"ing the @config (for the resource they sort) and then disables the sorter, if the given resource is full. Nice and extensible.

This didn't work until I found out, that disabling a sorter seems to mean it becomes an inverted sorter.
https://github.com/Anuken/Mindustry-Suggestions/issues/1070

[^1]: When coal is full, greater amounts of graphite are produced.
![image](https://user-images.githubusercontent.com/865652/179226184-25e0aed9-9f7d-4f5b-8e8b-4e4f5a7c97ab.png)
I know, this also solves my issue 
![image](https://user-images.githubusercontent.com/865652/179228885-e4ed142b-e156-4fbc-906a-94373296c235.png)

### Describe the changes you want to propose. Include possible alternatives.

I like the idea of being able to switch between a sorter and a router.
- Maybe use enabled/disable to turn off blocks.
- Use the "configure" property to toggle states (this could even have an interface similar to the sorter) and  also toggle the two states of overflow gates.

Don't take me too serious, I'm new to the game and don't know all the implications this change would mean.

Thanks again, I'm having a lot of fun :D

**Comment by schuhwerk** [comment]
True. My idea was to use the sorters as an interface for selecting the material that needs to be controlled.
So you don't have to do any programming if you want to add another material/sorter just add the sorter to the selection and you are done.
![image](https://user-images.githubusercontent.com/865652/179244982-50226400-da2d-4bb7-a5b7-d6c08e0a463d.png)

This is the code, maybe it helps understanding:
```mlog
set maxItems 13000
set i 0
getlink link i
sensor dead link @dead
jump 1 equal dead 1
sensor mytype link @type
jump 13 notEqual mytype @sorter
sensor material link @config # this gets the selected material from the router
sensor matInCore nucleus1 material
set isEnabled 0
jump 12 greaterThanEq matInCore maxItems
set isEnabled 1
control enabled link isEnabled 0 0 0
op add i i 1
jump 2 always x false
```
